ocean_lib/models/test/test_erc721_token.py

```diff
@@ -15,7 +15,7 @@
 )
```

Comment:
Many lines in this file (and others?) seem to have gone through `black` which converted them into 3-4 line statements, way harder to read. You should always ask yourself: how do I preserve readability?

For this file (and others?), here's one trick. Many of the lines have `consumer_wallet.address`. What you can do is add new function argument `consumer_addr`, which is defined in a method in conftest.py to equal consumer_wallet.address. Do the same for other often-used addresses: `publisher_wallet.address` and `another consumer_wallet.address`. In shortening that text, you should be able to convert back many of the 3-4 line chunks, back into a single line.

Reply:
It did some improvements. Thanks!

Reply:
I have created fixtures for all 3 used wallets in conftest.py: `publisher_addr`, `consumer_addr` and `anothor_consumer_addr`.